azure-pipeline.yml

```diff
@@ -1,5 +1,7 @@
 pool:
-  vmImage: windows-2019
+  vmImage: windows-2022
```

Comment:
Could we use windows-latest instead?

Reply:
Sure, the reason it is windows-2022 and not windows-latest was that when I started the pipeline windows-latest pointed to windows-2019.

Reply:
ok that makes sense.